test/fixtures/suite/read_from_stdin/test.bats

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bats
+
+function cmd_using_stdin {
+    # Just reading from stdin
+    while read -r -t 1 foo; do
```

Comment:
Each test here blocks for a second - could this be reduced to `read -r -t 0.001 foo`?

Reply:
I tried but a float as timeout is not supported on all Bash versions. The older ones only accept integers

Reply:
Argh, I wondered if that was the case. From my rudimentary testing looks like Bash 3 is the problem? As this adds three seconds I'm wondering if there's any way to avoid adding that time to every test run. We could set by `${BASH_VERSINFO}` perhaps?

Reply:
Good idea: bcce611